### DISCUSSION TITLE
Restore Downloader App

### DISCUSSION THREAD

**Discussion by Jaykob** [JonathanTreffler/backblaze-personal-wine-container#68] [Ideas]
Hi!
Thanks for this wonderful project!
Any plans on also supporting the Backblaze downloader app?
It could make a restore much simpler in case of a NAS backup, as the restore downloader could run on the NAS itself.

**Comment by JonathanTreffler** [comment]
Technically #71 is a duplicate to this, because you opened your discussion first, but there is a lot more information there, so I am closing this.

TLDR: Update to v1.6 and follow https://github.com/JonathanTreffler/backblaze-personal-wine-container/issues/71#issuecomment-1858456821 to use the restore app. I will try to make it work out of the box for v1.7.
